requirements-test.txt

```diff
@@ -0,0 +1,6 @@
+pytest
```

Comment:
We should specify the versions here

Reply:
None of our other models specific a version for the test requirements and having multiple versions could cause issues on our Jenkins infrastructure. Do you think this is really necessary?

Reply:
Nevermind thought this was the requirements.txt